source/_data/themes/ruyun.yml

```diff
@@ -0,0 +1,14 @@
+description: An ICP-friendly personal blog theme for Hexo
+link: https://github.com/ztygalaxy/hexo-theme-ruyun
+preview: https://tyzhang.top
+tags:
+  - hexo-theme
+  - Ruyun
```

Comment:
What is `Ruyun`?

Reply:
`Ruyun` is name of this theme, search-friendly.

Reply:
It has now been removed～